core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/MockApiService.scala

```diff
@@ -0,0 +1,90 @@
+import org.broadinstitute.dsde.rawls.spendreporting.SpendReportingService
+import org.broadinstitute.dsde.rawls.status.StatusService
+import org.broadinstitute.dsde.rawls.submissions.SubmissionsService
+import org.broadinstitute.dsde.rawls.user.UserService
+import org.broadinstitute.dsde.rawls.workspace.{MultiCloudWorkspaceService, WorkspaceService, WorkspaceSettingService}
+import org.broadinstitute.dsde.workbench.oauth2.OpenIDConnectConfiguration
+import org.scalatestplus.mockito.MockitoSugar.mock
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext
+import scala.language.postfixOps
+
+class MockApiService(
+  val userInfo: UserInfo =
+    UserInfo(RawlsUserEmail(""), OAuth2BearerToken("token"), 123, RawlsUserSubjectId("123456789876543212349")),
+  override val openIDConnectConfiguration: OpenIDConnectConfiguration = mock[OpenIDConnectConfiguration],
+  override val samDAO: SamDAO = mock[SamDAO],
+  override val submissionTimeout: FiniteDuration = mock[FiniteDuration],
+  override val workbenchMetricBaseName: String = "workbenchMetricBaseName-test",
+  override val batchUpsertMaxBytes: Long = 1028,
+  override val executionServiceCluster: ExecutionServiceCluster = mock[ExecutionServiceCluster],
+  override val appVersion: ApplicationVersion = mock[ApplicationVersion],
+  override val spendReportingConstructor: RawlsRequestContext => SpendReportingService = _ =>
+    mock[SpendReportingService],
+  override val billingProjectOrchestratorConstructor: RawlsRequestContext => BillingProjectOrchestrator = _ =>
+    mock[BillingProjectOrchestrator],
+  override val entityServiceConstructor: RawlsRequestContext => EntityService = _ => mock[EntityService],
+  override val genomicsServiceConstructor: RawlsRequestContext => GenomicsService = _ => mock[GenomicsService],
+  override val snapshotServiceConstructor: RawlsRequestContext => SnapshotService = _ => mock[SnapshotService],
+  override val statusServiceConstructor: () => StatusService = () => mock[StatusService],
+  override val methodConfigurationServiceConstructor: RawlsRequestContext => MethodConfigurationService = _ =>
+    mock[MethodConfigurationService],
+  override val submissionsServiceConstructor: RawlsRequestContext => SubmissionsService = _ => mock[SubmissionsService],
+  override val workspaceServiceConstructor: RawlsRequestContext => WorkspaceService = _ => mock[WorkspaceService],
+  override val multiCloudWorkspaceServiceConstructor: RawlsRequestContext => MultiCloudWorkspaceService = _ =>
+    mock[MultiCloudWorkspaceService],
+  override val workspaceSettingServiceConstructor: RawlsRequestContext => WorkspaceSettingService = _ =>
+    mock[WorkspaceSettingService],
+  override val bucketMigrationServiceConstructor: RawlsRequestContext => BucketMigrationService = _ =>
+    mock[BucketMigrationService],
+  override val userServiceConstructor: RawlsRequestContext => UserService = _ => mock[UserService]
+)(implicit val executionContext: ExecutionContext)
```

Comment:
The basic purpose of this is:
1. set up default mocks with *no* functionality, so you only need to specify the ones you care about
2. set up paths that reflect those in `RawlsApiService`, but without the parts that we don't care about, such as metrics and logging.